draft-ietf-jsonpath-base.md

```diff
@@ -1388,16 +1388,16 @@ Comparisons:
 | `$.arr != $.arr` | false | Array comparison |
 | `$.obj == 17` | false | Type mismatch |
 | `$.obj != 17` | true | Type mismatch |
-| `$.obj <= $.arr` | false | Objects and arrays are not ordered |
-| `$.obj < $.arr` | false | Objects and arrays are not ordered |
+| `$.obj <= $.arr` | false | Objects and arrays do not offer `<` comparison |
```

Comment:
Just wondering about the "offering" language. How about the following (and throughout)?
```suggestion
| `$.obj <= $.arr` | false | Objects and arrays cannot be compared using `<` |
```

Reply:
Well, this sounds like a factual statement, but it is not clear if that is a fundamental property (here, it actually pretty much is), or if it is a result of what we decided.

Reply:
We could s/offer/provide for/

Reply:
"provide for" is about the same as "offer" to my taste. How about "do not support"?

Reply:
(On the factual statement point, "not offering" is equally factual - it doesn't say that this spec doesn't offer.)

Reply:
The meaning is the same, the connotations aren't.  I'd like stick with "provide for" if you don't like "offer".

Reply:
It's not a big deal, just a matter of taste. Thanks for considering the alternatives. Merging...